Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -111,6 +118,15 @@ public struct OfflineMapAreasView: View {
         }
         .frame(maxWidth: .infinity)
     }
+    
+    /// Sets the closure to call when the map selection changes.
+    public func onMapSelectionChanged(
+        perform action: @escaping (_ newMap: Map) -> Void
+    ) -> Self {
```

Comment:
Passing a selection back to a parent view via a modifier like this is highly unusual. Look through SwiftUI's interface (command click on SwiftUI wherever it is imported) and search for "selection". It is (almost?) always passed via a binding taken as an initialization parameter. Is there a strong reason not to do that with this view?

Reply:
No, I removed the view modifier and added a binding parameter to the initializer. Thanks for the suggestion.

Reply:
You're welcome.